kart/tabular/working_copy/postgis.py

```diff
@@ -337,3 +343,14 @@ def _apply_meta_schema_json(self, sess, dataset, src_value, dest_value):
             sess.execute(
                 f"""ALTER TABLE {self.table_identifier(table)} ALTER COLUMN {self.quote(col.name)} TYPE {dest_type};"""
             )
+
+    def _check_for_unsupported_ds_types(self, sess, target_datasets, schema="public"):
+        for target_dataset in target_datasets:
+            table = target_datasets[target_dataset]
+            if table.is_spatial:
+                inspector = inspect(sess.bind)
```

Comment:
I suspect this involves some communication with the database (not sure though) - I'd init it outside the loop rather than for each dataset